packages/eslint-plugin-fiori-tools/package.json

```diff
@@ -25,8 +25,8 @@
     },
     "peerDependencies": {
         "eslint": ">=7",
-        "@typescript-eslint/eslint-plugin": "^5.59.0",
-        "@typescript-eslint/parser": "^5.59.0"
+        "@typescript-eslint/eslint-plugin": ">=5.59.0",
+        "@typescript-eslint/parser": ">=5.59.0"
```

Comment:
I am always confused with `~` `^` and `>=`. Could you please explain why this change?

Reply:
This is to address https://github.com/SAP/open-ux-tools/issues/1719#issuecomment-1976727749 
and set a minimum version of `"@typescript-eslint/*` so that it can be used with even newer major versions which are now available.

Reply:
Thanks.